docs/source/sections/webapp/releases.rst

```diff
@@ -3,6 +3,47 @@ Web-App Release Notes
 *********************
 
 
+* **June 18, 2018** 
+
+  **Please Note:** Work is underway to rework our cohort creation page to better differentiate between samples which are from image data vs. those which are not.
+  
+  **Issues resolved in sprint 27 as of 06/18/2018**
+  
+  New Enhancements
+   
+   - The ISB-CGC has enabled OncoPrint visualization tool for germline mutations from cBioPortal as a Workbook analysis. For more information please go `here <http://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/webapp/Workbooks.html#creating-and-saving-a-workbook>`_
+   - You are now able to view Radiology Images from TCIA data through the File Browser using the Osimis viewer. For more information please go here `here <http://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/webapp/OsimisWebViewer.html>`_
+   - Two new videos have been added to our video tutorials section. You can now learn how to sign up with a Google account and how to make a gene list easily. For more information please go here `here <https://isb-cgc.appspot.com/videotutorials/>`_
+   - The Dashboard has been upgraded to include a collapse feature for all panels (workbooks and cohorts are opened by default) and a direct link to the File Browser has been added to the Cohorts panel. 
+   - Under cohort creation by filters, the Molecular tab for TCGA data has been upgraded to combine multiple gene mutation filters. Filters can be combined using AND (requires all filters to be met for the data to be filtered) or OR (at least one criteria needs to be met for the data to be displayed).  
+   - The CSV download, Export to BigQuery, and Export to GCS feature has been added to the IGV Browser, Pathology Images, and the Radiology Images tab on the File Browser. 
+   - On the File Browser All files tab the clinical filter now displays the accurate count available for analysis. 
+   - The File Browser has been upgraded to now include the option of which columns to display and the ability to jump to any page. 
+   - The site menu has been improved to allow faster load times and better overall performance. Please Note that Workbooks must now be created from a data source (Cohorts, Variable lists, Gene & miRNA lists) or from the Workbook list page.
+   
+  Bug Fixes
+   
+   - When working on Firefox browser a violin plot will display the data plotted correctly when working on a Worksheet.
+   - A cohort with user uploaded data present and public data present in our system e.g TCGA data, the cohort details page for the selected filters panel will sort the filters by their appropriate program. 
+   - On the cohort creation - barcode upload a type-o in resulting cohorts counts has been addressed. The correct information is now displayed. 
+   - When trying to reload a stored Seq-Peek plot from a Workbook the previous gene selection is stored and the plot will automatically be loaded. 
```

Comment:
@madelyngreyes Sorry, I missed this before, but there was no typo in the counts. It was in the column headers. This makes it sound like the counts were incorrect, which is a potentially much larger issue than just columns being swapped. This should probably read 'On the cohort creation - barcode upload page the 'Samples' and 'Cases' column headers were sometimes swapped. This has been corrected.'